### PR TITLE
75325: Add note about configuring OOB Enternet port IP.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/snippets/xseries.rst [hunk 1]
```diff
@@ -191,6 +191,10 @@ Discover Out-of-Band Management IP Address
 Several methods are available to determine the IP address currently
```
Comment (Mrt134) [inline userguide/snippets/xseries.rst]:
```suggestion
**A functioning Out-of-Band Ethernet port IP address is required for remote support.**
It is recommended to immediately configure this IP address and test the network connection.
Several methods are available to determine the IP address currently
```

File: userguide/snippets/xseries.rst [hunk 2]
```diff
@@ -191,6 +191,10 @@ Discover Out-of-Band Management IP Address
...
 assigned to the X-Series Out-of-Band management interface.
 
+.. important:: **Configuring the OOB Ethernet port IP address and
```
Comment (Mrt134) [inline userguide/snippets/xseries.rst]:
```suggestion

```

File: userguide/snippets/xseries.rst [hunk 3]
```diff
@@ -191,6 +191,10 @@ Discover Out-of-Band Management IP Address
...
+   connecting it to the network is strongly recommended, and required
```
Comment (Mrt134) [inline userguide/snippets/xseries.rst]:
```suggestion

```

File: userguide/snippets/xseries.rst [hunk 4]
```diff
@@ -191,6 +191,10 @@ Discover Out-of-Band Management IP Address
...
+   for remote support**.
```
Comment (Mrt134) [inline userguide/snippets/xseries.rst]:
```suggestion

```

File: userguide/snippets/xseries.rst [hunk 5]
```diff
@@ -188,10 +188,13 @@ it into place.
 Discover Out-of-Band Management IP Address
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+**A functioning Out-of-Band Ethernet port IP address is required for remote support.**
+It is recommended to immediately configure this IP address and test the network connection.
```
Comment (wblock) [inline userguide/snippets/xseries.rst]: Please don't say "It is recommended".
```suggestion
Immediately configuring the OOB IP address and testing the network connection is strongly recommended.
```

**Pull request merged**